### PR TITLE
[Bug]: Fix Graph issues

### PR DESCRIPTION
**Pull request by nxanil**
@natabal 
Fix `PieGraph` `TreeGraph` `LineGraph`  issues.

### CODE REVIEW

File: GraphsV2/GaugeChart/index.js [hunk 1]
```diff
@@ -113,7 +113,7 @@ const GaugeChart = (props) => {
             <Label id="chart-value">
                 {parseInt(chartValue)}%
             </Label>
-            <PieChart width={width} height={height- 22} >
+            <PieChart width={width} height={height- 20} >
```
Comment (natabal) [inline GraphsV2/GaugeChart/index.js:116]: what does 20 represent?

File: GraphsV2/utils/helper.js [hunk 2]
```diff
@@ -224,16 +224,9 @@ export const scaleColor = (props, data, defaultColumn, setGraphColor) => {
...
 
-export const renderMessage = (props) => {
-    const {
-        data,
-        message,
-        id
-    } = props;
-    const messageClass = data && data.classes && data.classes.messageClass;
-
+export const renderMessage = (message, id) => {
```
Comment (natabal) [inline GraphsV2/utils/helper.js:227]: you are changing the signature of the method. Where is this used?

File: GraphsV2/utils/helper.js [hunk 3]
```diff
@@ -224,16 +224,9 @@ export const scaleColor = (props, data, defaultColumn, setGraphColor) => {
     return colorList;
 }
 
-export const renderMessage = (props) => {
-    const {
-        data,
-        message,
-        id
-    } = props;
-    const messageClass = data && data.classes && data.classes.messageClass;
```
Comment (natabal) [inline GraphsV2/utils/helper.js:233]: why do we remove the classes here?

File: GraphsV2/utils/helper.js [hunk 4]
```diff
@@ -224,16 +224,9 @@ export const scaleColor = (props, data, defaultColumn, setGraphColor) => {
...
-
+export const renderMessage = (message, id) => {
     return (
-        <div id={`${id}-message`} className={messageClass ? messageClass : "center-text"}>
+        <div id={`${id}-message`} className="center-text">
```
Comment (natabal) [inline GraphsV2/utils/helper.js:229]: My vote is to keep both classes: `center-text ${messageClass || ''}`

File: GraphsV2/utils/helper.js [hunk 5]
```diff
@@ -224,15 +224,16 @@ export const scaleColor = (props, data, defaultColumn, setGraphColor) => {
...
+        classes,
         message,
         id
     } = props;
-    const messageClass = data && data.classes && data.classes.messageClass;
+    const messageClass = classes && classes.messageClass;
 
-    return (
+     return (
         <div id={`${id}-message`} className={messageClass ? messageClass : "center-text"}>
```
Comment (natabal) [inline GraphsV2/utils/helper.js:237]: I would like to be able to add to the classes however keep the 'center-text'. Let's change to `center-text (messageClass || '')`

File: GraphsV2/GaugeChart/index.js (hunk 1 above)
Comment (natabal) [reply GraphsV2/GaugeChart/index.js:116]: I still do not understand why do we take 20 of the height?

**Comment by nxanil** [comment]
I still do not understand why do we take 20 of the height?

The height of the label is 10 and has margin 10 from the top that's why removed 20 from the Guage graph height.

**Pull request merged**